Gemfile

```diff
@@ -86,18 +96,18 @@ gem 'rb-readline'
 gem "paperclip", "~> 5.2.0"
 gem "ruby-openid", :require => "openid"
 gem "rack-openid"
-gem "authlogic", "~> 3.5.0"
+gem "authlogic", "4.1.0"
 gem 'authlogic-oid'
 gem "php-serialize", :require => "php_serialize"
-gem 'less-rails',   '~> 2.6'
+gem 'less-rails', '~> 3.0'
 gem 'progress_bar'
 gem 'impressionist'
 gem "recaptcha", require: "recaptcha/rails"
 
 # RESTful API Support
 gem 'grape'
 gem 'grape-entity'
-gem 'grape-swagger', '~> 0.25.3' # later versions require
+gem 'grape-swagger', '~> 0.28.0' # later versions require
```

Comment:
Lets remove the comment since we're past Ruby 2.2

Reply:
Yeah I will remove it